kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -75,9 +93,14 @@
         channelId: (state) => state.currentChannelId,
         available: (state) => state.pageState.content.available,
         extraFields: (state) => state.pageState.content.extra_fields,
+        nextcontent: (state) => state.pageState.content.next_content,
```

Comment:
Instead of having so many getters for the sub-fields of `pageState.content`, it would be cleaner to just have:

```javascript
content: (state) => state.pageState.content,
```

and then reference it with e.g.

```javascript
this.content.extra_fields
```

Also, need to watch out for names that are confusingly similar or conflicting. Here we have a getter called `this.nextcontent` and a method called `this.nextContent`.